bugout/journal.py

```diff
@@ -464,6 +465,37 @@ def create_tags(
         )
         return result
 
+    def create_entries_tags(
+        self,
+        token: Union[str, uuid.UUID],
+        journal_id: Union[str, uuid.UUID],
+        entries_tags: BugoutJournalEntriesTagsRequest,
+        auth_type: AuthType = AuthType.bearer,
+        **kwargs: Dict[str, Any],
+    ) -> BugoutJournalEntries:
+        tags_path = f"journals/{journal_id}/bulk_entries_tags"
+        headers = {
+            "Authorization": f"{auth_type.value} {token}",
+        }
+        json = {
+            "entries": [
+                {
+                    "journal_entry_id": str(entry.entry_id),
```

Comment:
Can we use one key everywhere, for example in `BugoutJournalEntryTagsRequest` you call it just `entry_id`